### ISSUE TITLE
Please update to mattermost-desktop 4.0.1

### ISSUE THREAD

**Issue by j1mc** [snapcrafters/mattermost-desktop#8]
mattermost-desktop v4.0.1 has been tagged in github. Please update to the latest release:  https://github.com/mattermost/desktop/releases/tag/v4.0.1

Thank you!

**Comment by j1mc** [comment]
The project developers have released the associated .deb files, as well (it looks like this project creates the snap by using the deb files). The .deb package is available at the same URL as above.

Also, is there a reason for why `snap info mattermost-desktop` shows version 4.0.0 as an "edge" release when it's already been out for a while?  Is there anything that needs to be done to move a release from "edge" through to beta, candidate, and then stable?  If I can assist, I'd be glad to help.

**Comment by popey** [comment]
Hi. Ideally we'd automate this, so it builds the latest available deb into the edge channel. That way we can easily promote it to stable as it will already be in the store. For some of the other snaps in the snapcrafters repo we have added logic (see [atom](https://github.com/snapcrafters/atom/blob/master/snap/snapcraft.yaml#L14)) which programmatically determines the version number. Would be good if we did the same for this snap too. It's 'easy' with atom because they have a redirector which always points to the latest deb we can consume. If mattermost had the same or similar, or we crafted some other shell magic, we could do that here too.

In the meantime anyone could propose a PR on the yaml to update the [version](https://github.com/snapcrafters/mattermost-desktop/blob/master/snap/snapcraft.yaml#L2) [number](https://github.com/snapcrafters/mattermost-desktop/blob/master/snap/snapcraft.yaml#L28). ;) ;)
